runtime/onert/odc/Quantizer.cc

```diff
@@ -47,16 +92,34 @@ int Quantizer::quantize(const char *in, const char *out, QuantizeType qtype)
 
   luci::CircleQuantizer quantizer;
   auto options = quantizer.options();
+  if (options == nullptr)
+    return 1;
+
+  // Fill quantization type param
+  fillQuantizeOptionParam(options, qtype);
+
+  // Additional phase for full quantization
+  if (full_quantize)
   {
-    options->enable(luci::CircleQuantizer::Options::Algorithm::QuantizeWeights);
+    // Fake quantization
+    options->enable(QuantizerOptions::Algorithm::QuantizeDequantizeWeights);
+    for (size_t idx = 0; idx < module->size(); ++idx)
+    {
+      auto graph = module->graph(idx);
 
-    using AlgorithmParameters = luci::CircleQuantizer::Options::AlgorithmParameters;
-    options->param(AlgorithmParameters::Quantize_input_model_dtype, "float32");
-    options->param(AlgorithmParameters::Quantize_output_model_dtype,
-                   qtype == QuantizeType::ODC_QTYPE_WO_I16_SYM ? "wo_int16" : "wo_int8");
-    options->param(AlgorithmParameters::Quantize_granularity, "channel");
+      // quantize the graph
+      quantizer.quantize(graph);
+    }
+
+    // TODO Record minmax by minmax-embedder
+    throw std::runtime_error{"Not implemented yet"};
```

Comment:
I have a question.
This line will cause the exception to always be thrown if `full_quantize` is `true`. Did you intended it?